f/ansible-playbook.json

```diff
@@ -660,11 +660,39 @@
     },
     "task": {
       "additionalProperties": true,
-      "not": {
-        "required": [
-          "block"
-        ]
-      },
+      "allOf": [
+        {
+          "not": {
+            "required": [
+              "block"
+            ]
+          }
+        },
+        {
+          "not": {
+            "required": [
+              "with_filetree"
+            ]
+          },
+          "title": "with_filetree was removed to with_community.general.filetree in 2.13"
```

Comment:
Why 2.13? These plugins moved to community.general along with all the rest in 2.10.

Reply:
True, corrected.